src/index.html

```diff
@@ -17,11 +17,466 @@
+              class="nav_link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div
+      class="product"
+      data-qa="card"
+    >
+          <div class="product__picture product__picture--modifier">
+          </div>
+
+          <div class="product__name product__name--modifier">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="product__code product__code--modifier">
+            Product code: 195434
+          </div>
+
+          <div class="product__rating">
+            <div class="product__rating-stars">
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+            </div>
+            <div class="product__rating-reviews">Reviews: 5</div>
+          </div>
+
+          <div class="product__price product__price--modifier">
+            <div class="product__price-text">Price:</div>
+            <div class="product__price-digital">$2,199</div>
+          </div>
+          <div
+            class="product__button"
+          >
+            <label for="buy"
+            data-qa="card-hover">
+              <form action="#">
+                <input
+                  type="button"
+                  value="buy"
+                  id="buy"
+                  name="buy"
+                  class="product__button-buy"
+                />
+              </form>
+            </label>
+          </div>
+        </div>
+
+        <div
+      class="product"
+
+    >
+          <div class="product__picture product__picture--modifier">
+          </div>
+
+          <div class="product__name product__name--modifier">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="product__code product__code--modifier">
+            Product code: 195434
+          </div>
+
+          <div class="product__rating">
+            <div class="product__rating-stars">
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+            </div>
+            <div class="product__rating-reviews">Reviews: 5</div>
+          </div>
+
+          <div class="product__price product__price--modifier">
+            <div class="product__price-text">Price:</div>
+            <div class="product__price-digital">$2,199</div>
+          </div>
+          <div
+            class="product__button"
+          >
+            <label for="buy2">
```

Comment:
Ensure that the `label` element is correctly associated with the input element using the `for` attribute. The `for` attribute should match the `id` of the input element.